custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
+        coordinator: OPNsenseDataUpdateCoordinator,
+        entity_description: SwitchEntityDescription,
+    ) -> None:
+        super().__init__(
+            config_entry=config_entry,
+            coordinator=coordinator,
+            entity_description=entity_description,
+        )
+        self._tracker: str = self._opnsense_get_tracker()
+        self._rule: Mapping[str, Any] | None = None
+        _LOGGER.debug(
+            f"[OPNsenseFilterSwitch init] Name: {self.name}, tracker: {self._tracker}"
```

Ruff

Logging statement uses f-string

(G004)